control_panel_api/aws.py

```diff
@@ -46,6 +46,20 @@ def put_bucket_logging(self, name, target_bucket, target_prefix):
                 }
             })
 
+    def put_bucket_tagging(self, name, tags):
+        """Put bucket tagging
+
+        :param name: Bucket name
+        :param tags: List of tags (key, value) e.g. [('buckettype', 'datawarehouse')]
+        :type name: str
+        :type tags: list(tuple)
+        """
+        self._do('s3', 'put_bucket_tagging',
+            Bucket=name,
+            Tagging={
+                'TagSet': [{'Key': k, 'Value': v} for k, v in tags]
```

Comment:
Maybe it's a problem for another day but as far as I can read from boto documentation the stuff inside `TagSet` needs to be a string. 

I'm not sure what could happen if they're not. We have few options here:
1) We don't do anything, after all we control the client code.
2) Just convert everything to string, e.g. calling `str(k)`/`str(v)`
3) We check the type and we blowup a `ValueError` or something like that (probably a bit too much)

`1)` may be alright but I think I'd prefer option `2)`, what do you reckon?

Reply:
I can `str()` it but also note we don't cast on any of the other calls